spec.html

```diff
@@ -36482,11 +36329,21 @@ <h1>CreateSetIterator ( _set_, _kind_ )</h1>
         <p>The abstract operation CreateSetIterator takes arguments _set_ and _kind_. This operation is used to create iterator objects for Set methods that return such iterators. It performs the following steps when called:</p>
         <emu-alg>
           1. Perform ? RequireInternalSlot(_set_, [[SetData]]).
-          1. Let _iterator_ be OrdinaryObjectCreate(%SetIteratorPrototype%, &laquo; [[IteratedSet]], [[SetNextIndex]], [[SetIterationKind]] &raquo;).
-          1. Set _iterator_.[[IteratedSet]] to _set_.
-          1. Set _iterator_.[[SetNextIndex]] to 0.
-          1. Set _iterator_.[[SetIterationKind]] to _kind_.
-          1. Return _iterator_.
+          1. Let closure be a new Abstract Closure with no parameters that captures _set_ and _kind_ and performs the following steps when called:
+            1. Let _index_ be 0.
+            1. Let _entries_ be the List that is _set_.[[SetData]].
+            1. NOTE: the number of elements of _entries_ mentioned below must be redetermined in each while loop.
```

Comment:
similarly, i'm not sure "in each while loop" is the best way to phrase this.

Reply:
```suggestion
            1. NOTE: the number of elements of _entries_ mentioned below must be redetermined every iteration.
```